bin/sentinel.py

```diff
@@ -60,6 +60,20 @@ def watchdog_check(dashd):
     printdbg("leaving watchdog_check")
 
 
+# ping dashd
+def sentinel_ping(dashd):
+    printdbg("in sentinel_ping")
+
+    # delete expired watchdogs
```

Comment:
I don't think it's a good idea to duplicate the wd expiration code here.  Needs a slightly better design to avoid code duplication, I think.